skrub/_table_vectorizer.py

```diff
@@ -368,13 +313,10 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
         ``len(transformers_)==len(transformers)+1``, otherwise
         ``len(transformers_)==len(transformers)``.
 
-    types_ : dict mapping of int to type
+    type_per_column_ : dict mapping of int to type
```

Comment:
I think we can come up with a better name but I don't have one :sweat_smile:  maybe something like `inferred_column_types_`, `autocast_dtypes_`, `detected_types_`, :thinking:

Reply:
`inferred_column_types_` sounds better